src/config.c

```diff
@@ -366,6 +498,20 @@ CONFIG_GETTER(getOnTimeout) {
   return sdsnew(TimeoutPolicy_ToString(config->requestConfigParams.timeoutPolicy));
 }
 
+// on-timeout
+int set_on_timeout(const char *name, int val, void *privdata,
```

Comment:
maybe have a generic set for the specific type instead of having it specific to `on_timeout`?

Reply:
Size of `*privdata` could be different (it is an enum config)